<ui version="4.0" >
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>388</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="mouseTracking" >
   <bool>false</bool>
  </property>
  <property name="windowTitle" >
   <string>FoldWatcher - by Dr.K@meleon</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="foldwatcher.qrc" >:/logo/logo/greenCube.png</iconset>
  </property>
  <property name="statusTip" >
   <string/>
  </property>
  <property name="styleSheet" >
   <string/>
  </property>
  <property name="iconSize" >
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <property name="toolButtonStyle" >
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="dockNestingEnabled" >
   <bool>false</bool>
  </property>
  <property name="dockOptions" >
   <set>QMainWindow::AnimatedDocks|QMainWindow::ForceTabbedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac" >
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>753</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFolding_Home" >
    <property name="title" >
     <string>Folding@Home</string>
    </property>
    <widget class="QMenu" name="menuProgram" >
     <property name="windowIcon" >
      <iconset>../../../.designer/backup</iconset>
     </property>
     <property name="statusTip" >
      <string/>
     </property>
     <property name="title" >
      <string>Program</string>
     </property>
     <addaction name="separator" />
     <addaction name="actionStart" />
     <addaction name="actionStop" />
    </widget>
    <widget class="QMenu" name="menuClients" >
     <property name="title" >
      <string>Clients</string>
     </property>
     <addaction name="actionAddClient" />
     <addaction name="actionRemoveClient" />
    </widget>
    <addaction name="menuProgram" />
    <addaction name="menuClients" />
    <addaction name="separator" />
    <addaction name="actionSaveConfig" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuOptions" >
    <property name="title" >
     <string>Options</string>
    </property>
    <addaction name="actionPreferences" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionSourceForge" />
    <addaction name="actionHomepage" />
    <addaction name="separator" />
    <addaction name="actionContact" />
    <addaction name="separator" />
    <addaction name="actionAbout" />
   </widget>
   <widget class="QMenu" name="menuResources" >
    <property name="title" >
     <string>Resources</string>
    </property>
    <addaction name="actionFoldingWebsite" />
    <addaction name="actionFoldingForum" />
   </widget>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>Tools</string>
    </property>
    <addaction name="actionPersonalStats" />
    <addaction name="actionTeamStats" />
    <addaction name="separator" />
    <addaction name="actionMoleculeViewer" />
    <addaction name="separator" />
   </widget>
   <addaction name="menuFolding_Home" />
   <addaction name="menuView" />
   <addaction name="menuOptions" />
   <addaction name="menuTools" />
   <addaction name="menuResources" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="foldingToolbar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable" >
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart" />
   <addaction name="actionStop" />
   <addaction name="separator" />
   <addaction name="actionAddClient" />
   <addaction name="actionRemoveClient" />
  </widget>
  <widget class="QToolBar" name="toolBar_2" >
   <property name="windowTitle" >
    <string>toolBar_2</string>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionPersonalStats" />
   <addaction name="actionTeamStats" />
  </widget>
  <widget class="QToolBar" name="toolBar_3" >
   <property name="windowTitle" >
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionSaveConfig" />
   <addaction name="actionPreferences" />
   <addaction name="separator" />
   <addaction name="actionMoleculeViewer" />
  </widget>
  <widget class="QToolBar" name="toolBar_4" >
   <property name="windowTitle" >
    <string>toolBar_4</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbout" />
   <addaction name="actionExit" />
  </widget>
  <widget class="QToolBar" name="toolbarLogs" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="actionStart" >
   <property name="icon" >
    <iconset resource="foldwatcher.qrc" >:/icons/icons/gpm-brightness-lcd.png</iconset>
   </property>
   <property name="text" >
    <string>Start</string>
   </property>
   <property name="statusTip" >
    <string>Start all Folding@Home Clients</string>
   </property>
  </action>
  <action name="actionStop" >
   <property name="icon" >
    <iconset resource="foldwatcher.qrc" >:/icons/icons/stock_stop.png</iconset>
   </property>
   <property name="text" >
    <string>Stop</string>
   </property>
   <property name="statusTip" >
    <string>Stop all Folding@Home Clients</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="icon" >
    <iconset resource="foldwatcher.qrc" >:/icons/icons/stock_exit.png</iconset>
   </property>
   <property name="text" >
    <string>Exit</string>
   </property>
   <property name="statusTip" >
    <string>Exit FoldWatcher</string>
   </property>
  </action>
  <action name="actionAddClient" >
   <property name="icon" >
    <iconset resource="foldwatcher.qrc" >:/icons/icons/add.png</iconset>
   </property>
   <property name="text" >
    <string>Add a client</string>
   </property>
   <property name="statusTip" >
    <string>Add a client for monitoring</string>
   </property>
  </action>
  <action name="actionRemoveClient" >
   <property name="icon" >
    <iconset resource="foldwatcher.qrc" >:/icons/icons/remove.png</iconset>
   </property>
   <property name="text" >
    <string>Remove a client</string>
   </property>
   <property name="statusTip" >
    <string>Remove a client being monitored</string>
   </property>
  </action>
  <action name="actionPreferences" >
   <property name="icon" >
    <iconset resource="foldwatcher.qrc" >:/icons/icons/xfce-system.png</iconset>
   </property>
   <property name="text" >
    <string>Preferences</string>
   </property>
   <property name="statusTip" >
    <string>Modify FoldWatcher's options</string>
   </property>
  </action>
  <action name="actionSourceForge" >
   <property name="icon" >
    <iconset resource="foldwatcher.qrc" >:/icons/icons/gnome-globe.png</iconset>
   </property>
   <property name="text" >
    <string>FoldWatcher @ SourceForge.net</string>
   </property>
   <property name="statusTip" >
    <string>Visit SourceForge.net page for FoldWatcher</string>
   </property>
  </action>
  <action name="actionHomepage" >
   <property name="icon" >
    <iconset resource="foldwatcher.qrc" >:/icons/icons/stock_home.png</iconset>
   </property>
   <property name="text" >
    <string>FoldWatcher Homepage</string>
   </property>
   <property name="statusTip" >
    <string>Visit FoldWatcher Homepage</string>
   </property>
  </action>
  <action name="actionContact" >
   <property name="icon" >
    <iconset resource="foldwatcher.qrc" >:/icons/icons/stock_mail-reply.png</iconset>
   </property>
   <property name="text" >
    <string>Contact the author</string>
   </property>
   <property name="statusTip" >
    <string>E-mail the author</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="icon" >
    <iconset resource="foldwatcher.qrc" >:/icons/icons/system-help.png</iconset>
   </property>
   <property name="text" >
    <string>About...</string>
   </property>
   <property name="statusTip" >
    <string>About FoldWatcher</string>
   </property>
  </action>
  <action name="actionFoldingWebsite" >
   <property name="icon" >
    <iconset resource="foldwatcher.qrc" >:/icons/icons/gnome-globe.png</iconset>
   </property>
   <property name="text" >
    <string>Folding@Home Website</string>
   </property>
   <property name="statusTip" >
    <string>Visit main Folding@Home website</string>
   </property>
  </action>
  <action name="actionFoldingForum" >
   <property name="icon" >
    <iconset resource="foldwatcher.qrc" >:/icons/icons/gnome-globe.png</iconset>
   </property>
   <property name="text" >
    <string>Folding@Home Forum</string>
   </property>
   <property name="iconText" >
    <string>Folding@Home Forum</string>
   </property>
   <property name="toolTip" >
    <string>Folding@Home Forum</string>
   </property>
   <property name="statusTip" >
    <string>Visit Folding@Home forum</string>
   </property>
  </action>
  <action name="actionUnitinfo_txt" >
   <property name="icon" >
    <iconset resource="foldwatcher.qrc" >:/icons/icons/stock_text_justify.png</iconset>
   </property>
   <property name="text" >
    <string>unitinfo.txt</string>
   </property>
   <property name="statusTip" >
    <string>Show "unitinfo.txt" for current client</string>
   </property>
  </action>
  <action name="actionFAHlog_txt" >
   <property name="icon" >
    <iconset resource="foldwatcher.qrc" >:/icons/icons/stock_text_justify.png</iconset>
   </property>
   <property name="text" >
    <string>FAHlog.txt</string>
   </property>
   <property name="statusTip" >
    <string>Show "FAHlog.txt" for current client</string>
   </property>
  </action>
  <action name="actionCurrent_xyz" >
   <property name="icon" >
    <iconset resource="foldwatcher.qrc" >:/icons/icons/stock_text_justify.png</iconset>
   </property>
   <property name="text" >
    <string>current.xyz</string>
   </property>
   <property name="statusTip" >
    <string>Show "current.xyz" for current client</string>
   </property>
  </action>
  <action name="actionFoldWatcher_Log" >
   <property name="icon" >
    <iconset resource="foldwatcher.qrc" >:/icons/icons/stock_autopilot.png</iconset>
   </property>
   <property name="text" >
    <string>FoldWatcher Log</string>
   </property>
   <property name="statusTip" >
    <string>Show FoldWatcher log</string>
   </property>
  </action>
  <action name="actionMoleculeViewer" >
   <property name="icon" >
    <iconset resource="foldwatcher.qrc" >:/icons/icons/applications-science.png</iconset>
   </property>
   <property name="text" >
    <string>Molecule Viewer</string>
   </property>
   <property name="statusTip" >
    <string>Graphical 3D representation of protein being folded</string>
   </property>
  </action>
  <action name="actionPersonalStats" >
   <property name="icon" >
    <iconset resource="foldwatcher.qrc" >:/icons/icons/stock_person.png</iconset>
   </property>
   <property name="text" >
    <string>Retrieve Personal Stats</string>
   </property>
   <property name="statusTip" >
    <string>Retrieve personal statistics</string>
   </property>
  </action>
  <action name="actionTeamStats" >
   <property name="icon" >
    <iconset resource="foldwatcher.qrc" >:/icons/icons/stock_people.png</iconset>
   </property>
   <property name="text" >
    <string>Retrieve Team Stats</string>
   </property>
   <property name="statusTip" >
    <string>Retrieve team statistics</string>
   </property>
  </action>
  <action name="actionProtein_information" >
   <property name="icon" >
    <iconset resource="foldwatcher.qrc" >:/icons/icons/messagebox_info.png</iconset>
   </property>
   <property name="text" >
    <string>Protein information</string>
   </property>
   <property name="statusTip" >
    <string>Retrieve information on protein being folded</string>
   </property>
  </action>
  <action name="actionSaveConfig" >
   <property name="icon" >
    <iconset resource="foldwatcher.qrc" >:/icons/icons/document-save.png</iconset>
   </property>
   <property name="text" >
    <string>Save configuration</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="foldwatcher.qrc" />
 </resources>
 <connections/>
</ui>
